README.nfsroot.md

```diff
@@ -1,22 +1,39 @@
 # poky-nfsroot
 
 Example steps to use:
+
+Prepare your `local.conf`:
+
 ```
 echo 'INHERIT += "nfsroot"' >> $BUILDDIR/conf/local.conf
 echo 'IMAGE_FSTYPES:append = " tar.gz"' >> $BUILDDIR/conf/local.conf
-bitbake meta-ide-support
```

Comment:
For the record: no longer needed since dea0b2d2a9345e6df771aaaf2f9f2796343d6e30.